x-codemod unable to migrate any files + missing renameCSSClasses in "preset-safe"

### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Steps to reproduce 🕹

Link to live example:


Steps:
1. npx @mui/x-codemod v6.0.0/preset-safe .\PMP\src\




### Current behavior 😯

```
Error: Cannot find module 'packages/x-codemod/src/util/renameCSSClasses'
Require stack:
- C:\Users\MyComputerUser\AppData\Roaming\npm-cache\_npx\13512\node_modules\@mui\x-codemod\v6.0.0\data-grid\rename-linkOperators-logicOperators\index.js
- C:\Users\MyComputerUser\AppData\Roaming\npm-cache\_npx\13512\node_modules\@mui\x-codemod\v6.0.0\data-grid\preset-safe\index.js
- C:\Users\MyComputerUser\AppData\Roaming\npm-cache\_npx\13512\node_modules\@mui\x-codemod\v6.0.0\preset-safe\index.js
- C:\Users\MyComputerUser\AppData\Roaming\npm-cache\_npx\13512\node_modules\@mui\x-codemod\node_modules\jscodeshift\src\Worker.js
```

Results:
0 errors
0 unmodified
0 skipped
0 ok
Time elapsed: 3.667seconds



### Expected behavior 🤔

No errors and able to transform v5.15.5 <DataGridPro params

Note: Is this tool only searching for <DataGrid /> components and not <DataGridPro />?

### Context 🔦

Quick convert old v5 props to v6

### Your environment 🌎

<details>
  <summary><code>npx @mui/envinfo</code></summary>

```
npx: installed 2 in 4.216s

  System:
    OS: Windows 10 10.0.19044
  Binaries:
    Node: 14.18.3 - C:\Program Files\nodejs\node.EXE        
    Yarn: 1.22.19 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 6.14.15 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Chrome: Not Found
    Edge: Spartan (44.19041.1266.0), Chromium (110.0.1587.46)
  npmPackages:
    @emotion/react:  11.9.0
    @emotion/styled:  11.8.1
    @mui/base:  5.0.0-alpha.79
    @mui/icons-material:  5.6.2
    @mui/lab:  5.0.0-alpha.80
    @mui/material:  5.6.4
    @mui/private-theming:  5.6.2
    @mui/styled-engine:  5.6.1
    @mui/system:  5.6.4
    @mui/types:  7.1.3
    @mui/utils:  5.6.1
    @mui/x-data-grid:  6.0.0-beta.3
    @mui/x-data-grid-pro:  6.0.0-beta.3
    @mui/x-date-pickers:  6.0.0-beta.3
    @mui/x-license-pro:  6.0.0-beta.3
    @types/react:  17.0.20
    react:  17.0.2
    react-dom:  17.0.2
    typescript:  4.4.2

```
</details>


### Order ID 💳 (optional)

_No response_

--- a/packages/x-codemod/src/v6.0.0/data-grid/rename-linkOperators-logicOperators/index.ts
+++ b/packages/x-codemod/src/v6.0.0/data-grid/rename-linkOperators-logicOperators/index.ts
@@ -1,4 +1,4 @@
-import renameCSSClasses from 'packages/x-codemod/src/util/renameCSSClasses';
+import renameCSSClasses from '../../../util/renameCSSClasses';
 import { JsCodeShiftAPI, JsCodeShiftFileInfo } from '../../../types';
 import renameIdentifiers, { PreRequisiteUsage, matchImport } from '../../../util/renameIdentifiers';
 

